src/cursor/cur_backup_incr.c

```diff
@@ -83,18 +83,19 @@ __curbackup_incr_blkmod(WT_SESSION_IMPL *session, WT_BTREE *btree, WT_CURSOR_BAC
         WT_ERR_NOTFOUND_OK(__wt_config_subgets(session, &v, "rename", &b), true);
         if (ret == 0 && b.val)
             F_SET(cb, WT_CURBACKUP_RENAME);
-        else
+        else {
```

Comment:
I didn't clearly understand the reason behind not traversing the blocks information. Can you add a comment to explain it?

Reply:
added a comment!